democracy/models/comment.py

```diff
@@ -56,11 +57,25 @@ def parent_id(self):
         """
         return getattr(self, "%s_id" % self.parent_field, None)
 
+    def _detect_lang(self):
+        try:
+            candidates = detect_langs(self.content)
+            for candidate in candidates:
+                if candidate.lang in [trans.language_code for trans in self.section.translations.all()]:
```

Comment:
Actually, let's change this to `settings.PARLER_LANGUAGES`, because we should actually allow (and already do have) comments in all supported languages even though the hearing only is in Finnish.